@here/harp-datasource-protocol/lib/Expr.ts

```diff
@@ -449,6 +559,33 @@ export class CaseExpr extends Expr {
     }
 }
 
+/**
+ * Expressions that represent literal/immediate values.
+ *
+ * @hidden
+ */
+export type LiteralExpr =
```

Comment:
please introduced a base abstract class `LiteralExpr` that extends `Expr` instead.

Reply:
done